mmstudio/src/main/java/org/micromanager/display/internal/animate/AnimationController.java

```diff
@@ -309,8 +311,8 @@ public void run() {
             @Override
             public void run() {
                synchronized (AnimationController.this) {
+                  // TODO: should this be done at all?
                   listeners_.fire().animationAcknowledgeDataPosition(newPosition);
-                 
```

Comment:
Not sure at this point, but I think originally this was part of the mechanism to prevent the last frame from being missing from the display window scrollbar range.

Reply:
Yes, this may be part of one of the code paths updating the sliders when displayImage is not called.